lib/faulty/circuit.rb

```diff
@@ -156,6 +156,13 @@ def finalize
       end
     end
 
+    # @return [String] Text representation of the circuit
+    def inspect
+      interested_opts = %i(cool_down evaluation_window rate_threshold sample_threshold)
```

Comment:
These happen to be the same as `Faulty::Circuit::Options#for_storage` is that the intention or should we keep a separate list here?

Reply:
Circuit has these options

```
[:cache_expires_in, :cache_refreshes_after, :cache_refresh_jitter, :cool_down, :evaluation_window, :rate_threshold, :sample_threshold, :errors, :error_module, :exclude, :cache, :notifier, :storage, :registry]
```

which storage and registry will be huge string when we connect to a real Redis.

We can also change the `inspect` for `storage` and `registry` to only print relevant information.

So I think we only need to list circuit options